From efb0986ab203346252fcdcfa4859985750cb89d4 Mon Sep 17 00:00:00 2001
From: Zhixiong Chi <zhixiong.chi@windriver.com>
Date: Wed, 1 Apr 2020 02:26:55 -0700
Subject: [PATCH] pinctrl: freescale: imx: Drop the unused variables

Commit 4db927320e72 ("driver: pincrtl: pinctrl-imx: remove attaching
dev for regmap") delete attaching related code, so the two variables
are not used any more.
We remove the two varibales to avoid the compiling warnings now.

Signed-off-by: Zhixiong Chi <zhixiong.chi@windriver.com>
---
 drivers/pinctrl/freescale/pinctrl-imx.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/drivers/pinctrl/freescale/pinctrl-imx.c b/drivers/pinctrl/freescale/pinctrl-imx.c
index b04edc22dad7..2d4adb275acb 100644
--- a/drivers/pinctrl/freescale/pinctrl-imx.c
+++ b/drivers/pinctrl/freescale/pinctrl-imx.c
@@ -10,7 +10,6 @@
 #include <linux/err.h>
 #include <linux/init.h>
 #include <linux/io.h>
-#include <linux/mfd/syscon.h>
 #include <linux/of.h>
 #include <linux/of_device.h>
 #include <linux/of_address.h>
@@ -19,7 +18,6 @@
 #include <linux/pinctrl/pinctrl.h>
 #include <linux/pinctrl/pinmux.h>
 #include <linux/slab.h>
-#include <linux/regmap.h>
 
 #include "../core.h"
 #include "../pinconf.h"
@@ -675,13 +673,11 @@ static void imx_free_resources(struct imx_pinctrl *ipctl)
 int imx_pinctrl_probe(struct platform_device *pdev,
 		      const struct imx_pinctrl_soc_info *info)
 {
-	struct regmap_config config = { .name = "gpr" };
 	struct device_node *dev_np = pdev->dev.of_node;
 	struct pinctrl_desc *imx_pinctrl_desc;
 	struct device_node *np;
 	struct imx_pinctrl *ipctl;
 	struct resource *res;
-	struct regmap *gpr;
 	int ret, i;
 
 	if (!info || !info->pins || !info->npins) {
-- 
2.17.1

